tests/sentry/integrations/github/test_integration.py

```diff
@@ -145,6 +145,34 @@ def test_basic_flow(self):
         )
         assert oi.config == {}
 
+    @responses.activate
+    def test_github_installed_on_another_org(self):
```

Comment:
You should also verify that deleting the other integration would allow the install to go through.